scripts/import_from_ena_legacy.py

```diff
@@ -0,0 +1,849 @@
+        es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'sex', 'sex')
+        es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'breed', 'breed')
+        if species_key:
+            es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'organism', species_key)
+    else:  # specimen
+        es_type = 'specimen'
+        if is_ebi_record:
+            if 'organism part' in data['characteristics']:
+                es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'organismPart',
+                                                          'organism part', 'specimenFromOrganism')
+                es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'cellType', 'organism part')
+            elif 'cell type' in data['characteristics']:
+                es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'cellType', 'cell type')
+            es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'developmentalStage',
+                                                      'developmental stage', 'specimenFromOrganism')
+        else:  # from NCBI or DDBJ
+            if 'tissue' in data['characteristics']:
+                es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'cellType', 'tissue')
+                es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'organismPart',
+                                                          'tissue', 'specimenFromOrganism')
+            elif 'cell type' in data['characteristics']:
+                es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'cellType', 'cell type')
+            es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'developmentalStage',
+                                                      'development stage', 'specimenFromOrganism')
+
+        if species_key:
+            es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'organism', species_key, 'organism')
+        es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'sex', 'sex', 'organism')
+        es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'breed', 'breed', 'organism')
+        es_doc, found_fields = extract_field_info(data, es_doc, found_fields, 'breed', 'strain', 'organism')
+
+    if 'release' in data:
+        es_doc['releaseDate'] = parse_date(data['release'])
+    else:
+        es_doc['releaseDate'] = None
+
+    if 'update' in data:
+        es_doc['updateDate'] = parse_date(data['update'])
+    else:
+        es_doc['updateDate'] = None
+    # as no project = FAANG (otherwise already dealt with by import biosample), the standard is always basic
+    es_doc['standardMet'] = 'Legacy (basic)'
+    # animal only gets assigned during dealing with the relationship derived from
+    try:
+        animal
+    except NameError:
+        pass
```

Comment:
Need to do something with exception, it will be great to log it

Reply:
google result suggests to use this way to test whether a variable is defined, here to check whether animal is defined. If so, assign the derivedFrom attribute

Reply:
add relevant link as comment in 27a573a